pipeline/matcher/scripts/getMatcherGraph.ts

```diff
@@ -0,0 +1,191 @@
+import { existsSync, mkdirSync, writeFileSync } from 'fs';
+import fetch from 'node-fetch';
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb';
+import { DynamoDBDocument, QueryCommandInput } from '@aws-sdk/lib-dynamodb';
+import { getCreds } from '@weco/ts-aws/sts';
+import { digraph, INode, RootCluster, toDot } from 'ts-graphviz';
+import { exec } from 'child_process';
+import { SourceWork } from './models';
+import { getAttributes } from './graphAttributes';
+
+type UserInput = {
+  canonicalId: string;
+  pipelineDate: string;
+}
+
+type ElasticConfig = {
+  worksIndex: string;
+  imagesIndex: string;
+}
+
+
+async function getInput(): Promise<UserInput> {
+  let id: string;
+  if (process.argv.length != 3) {
```

Comment:
Never the double equals!
```suggestion
  if (process.argv.length !== 3) {
```